src/components/CardOnList/CardOnList.jsx

```diff
@@ -0,0 +1,89 @@
+import { Component } from 'react';
+import './style.scss'
+
+class CardOnList extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countPick: this.props.count,
+      amount: this.props.amount,
+    }
+  }
+
+  addProduct = async () => {
+    await this.setState(prevState => ({countPick: prevState.countPick + 1}));
+    await this.setState(prevState => ({amount: prevState.amount - 1}));
+    this.props.calculateValueBasket(this.props.cost);
+    this.props.rememberCount(this.props.id, this.state.countPick, this.state.amount);
+  }
+
+  deleteProduct = async () => {
+    await this.setState(prevState => ({countPick: prevState.countPick - 1}));
+    await this.setState(prevState => ({amount: prevState.amount + 1}));
+    this.props.calculateValueBasket(-this.props.cost);
```

Comment:
you can't do that
most likely you should have some condition in the calculateValueBasket function, depending on which you either increase your amount or decrease it